Basic structure and style for Homescreen and Header

juliakimseohyeon opened the pull request:
Hi team, when you have a moment please take a look at the Homescreen and Header. Thank you 😊

juliakimseohyeon committed:
Fix global colour values

juliakimseohyeon committed:
Fix margin and padding issues

juliakimseohyeon committed:
Fix icon size issues in Header

juliakimseohyeon committed:
Stylesheet update

juliakimseohyeon commented:
Related to Ticket #21

nicmart-dev commented:
@juliakimseohyeon great job matching with the mockup!

A few notes:

1. As login will be taken opportunistically, I suggest to hide the login icon behind a feature toggle (or comment out the associated code) and also same for the saved recipes section.
2. small enhancement, you used a logout instead of login icon.
3. Add `.DS_Store` to `.gitignore`

nicmart-dev reviewed (changes requested):
@juliakimseohyeon great job matching with the mockup!

A few notes:
1. As login will be taken opportunistically, I suggest to hide the login icon behind a feature toggle (or comment out the associated code) and also same for the saved recipes section.
2. small enhancement, you used a logout instead of login icon.
3. Add `.DS_Store` to `.gitignore`

juliakimseohyeon committed:
Add French and Chinese versions of the text to Home screen

juliakimseohyeon committed:
Add function in language button to switch languages on click

juliakimseohyeon committed:
Create footer structure and style

juliakimseohyeon committed:
Add different languages to Footer

nicmart-dev reviewed (changes requested):
Nice job Julia on footer and change language!

A few comments:
- I still see a few `.DS_Store` files in the PR.
- When changing language, some text is still in English (ie. Hungry Chef and "Show me" button, and I think app name itself should be translated). Also when clicking that button in another language, language reverts back to English
![image](https://github.com/user-attachments/assets/87829689-be3e-4e43-8e01-f80927fef375)

juliakimseohyeon committed:
Add FormattedMessages of App name, username, and home CTA button in Home and Header components

juliakimseohyeon committed:
Add dummy submit function to Home page

juliakimseohyeon committed:
Remove cached .DS_Store from repository and ignore it

juliakimseohyeon committed:
Add navigate function to form submission on Home page

juliakimseohyeon committed:
Add multilingual versions of recipe name in recipe data

juliakimseohyeon commented:
Hi @nicmart-dev, thanks for pointing those out! I've removed the cached `.DS_Store` files and added a onSubmit function to the form on the home page. This should prevent the language from reverting back to English now.

juliakimseohyeon committed:
Remove all .DS_Store files from repository and ignore them

nicmart-dev commented on tailwind.config.js:
I think this file at the root should be removed, and only the one in client folder remain.

nicmart-dev commented on package.json:
I think the changes should be undone, as those dependencies belong in the client folder, not at the root.

nicmart-dev reviewed (commented):
Some minor comments regarding files at the root.

juliakimseohyeon committed:
Merge branch 'develop' into feature/frontend-homescreen
